language-server/langserver/src/main/java/edu/umn/cs/melt/oberon0/langserver/Oberon0LanguageServer.java

```diff
@@ -0,0 +1,170 @@
+package edu.umn.cs.melt.oberon0.langserver;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+
+import org.eclipse.lsp4j.FileOperationFilter;
+import org.eclipse.lsp4j.FileOperationOptions;
+import org.eclipse.lsp4j.FileOperationPattern;
+import org.eclipse.lsp4j.FileOperationsServerCapabilities;
+import org.eclipse.lsp4j.InitializeParams;
+import org.eclipse.lsp4j.InitializeResult;
+import org.eclipse.lsp4j.MessageParams;
+import org.eclipse.lsp4j.MessageType;
+import org.eclipse.lsp4j.SemanticTokensLegend;
+import org.eclipse.lsp4j.SemanticTokensServerFull;
+import org.eclipse.lsp4j.SemanticTokensWithRegistrationOptions;
+import org.eclipse.lsp4j.ServerCapabilities;
+import org.eclipse.lsp4j.TextDocumentSyncKind;
+import org.eclipse.lsp4j.WorkspaceFolder;
+import org.eclipse.lsp4j.WorkspaceServerCapabilities;
+import org.eclipse.lsp4j.services.LanguageClient;
+import org.eclipse.lsp4j.services.LanguageClientAware;
+import org.eclipse.lsp4j.services.LanguageServer;
+import org.eclipse.lsp4j.services.TextDocumentService;
+import org.eclipse.lsp4j.services.WorkspaceService;
+
+import com.google.gson.JsonObject;
+
+import edu.umn.cs.melt.Oberon0.core.concreteSyntax.NModule_c;
+import edu.umn.cs.melt.lsp4jutil.Util;
+
+public class Oberon0LanguageServer implements LanguageServer, LanguageClientAware {
+
+  private Oberon0LanguageService service;
```

Comment:
Style note, it looks like you are using a mix of spaces and tabs for indentation.  We don't really have a prefered style for which one to use, but mixing them makes this file kinda hard to read.

Reply:
Done in [805e115](https://github.com/melt-umn/Oberon0/pull/14/commits/805e1155bddf92a6c4eabd0f412a8f6a6222e774)